src/app/projects/project-details/project-details-build/project-details-build.component.html

```diff
@@ -1,17 +1,12 @@
-<p-table
-  class="build-list"
-  [value]="project?.buildHistory"
-  [lazy]="true"
-  (onLazyLoad)="loadBuildsLazy($event)"
-  [paginator]="true"
-  [rows]="rowsCount"
-  [totalRecords]="buildsTotalCount">
+<p-table class="build-list" [value]="project?.buildHistory" [lazy]="true" (onLazyLoad)="loadBuildsLazy($event)"
+  [paginator]="true" [rows]="rowsCount" [totalRecords]="buildsTotalCount">
```

Comment:
Guessing this is because of some automatic formatter. Suggest tinkering with its settings because this is less readable IMO

Reply:
Found a setting that does wrap them, but it doesn't seem to be very configurable.

If you and @fredx30 like the format in `project-details-build.component.html` and `project-details.component.html` from the most recent change then I will keep using it. If not, then I can try to figure out how to get it to be like how the code was before.

Reply:
With the idea of "keep the PR as scoped as possible", I think you should just disable it. If we settle on that attribute format then we can open a different PR that applies it throughout before we enable our formatters.

These attribute changes are quite verbose and bloat the diffs a lot.

Reply:
Reverted formatting changes in e091874